assets/src/edit-story/utils/useWindowSize.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+const useWindowSize = () => {
+	const [ width, setWidth ] = useState( 0 );
+	const [ height, setHeight ] = useState( 0 );
+
+	useEffect( () => {
+		setWidth( window.innerWidth );
+		setHeight( window.innerHeight );
+
+		const onResize = () => {
+			setWidth( window.innerWidth );
+			setHeight( window.innerHeight );
+		};
+
+		window.addEventListener( 'resize', onResize );
+
+		return () => {
+			window.removeEventListener( 'resize', onResize );
+		};
+	}, [] );
+
+	return [ width, height ];
```

Comment:
Is an array the best interface here? Wouldn't an object make more sense? Currently it's a little weird if you only need `height`. But then they should probably be renamed to `windowWidth` and `windowHeight` at least in the returned object.

Reply:
One could use `[ , height ]` but yeah this could be changed to an object. At the moment the height is not really needed though 🙂